src/Structures/Nav.php

```diff
@@ -74,7 +74,7 @@ public function collections($collections = null)
     public function showUrl($params = [])
     {
         return cp_route('navigation.show', array_merge($params, [
-            'navigation' => $this->handle(),
+            'nav' => $this->handle(),
```

Comment:
If we're required to change the name of arguments, this is more of a breaking change than I'd like.

Reply:
We're not route model binding this one, and Laravel controller methods handle it fine with any variable/argument name.  It's just renaming the segment, because that's how `Route::resource()` works.  Curious how this is breaking for people?

Reply:
If someone was referencing `navigation`, it would be broken:

```
cp_route('navigation.show', ['navigation' => 'links'])
```

Reply:
Gotcha! Fixed this one.